example/src/Examples/NativeBottomTabs.tsx

```diff
@@ -9,10 +9,27 @@ const Tab = createNativeBottomTabNavigator();
 
 function NativeBottomTabs() {
   return (
-    <Tab.Navigator>
+    <Tab.Navigator
+      screenListeners={{
+        tabLongPress: (data) => {
+          console.log(
+            '### JS Navigator level TabLongPress called',
+            JSON.stringify(data)
+          );
+        },
+      }}
+    >
```

Comment:
Could you test the integration like this? (From JS bottom tabs docs)

```
React.useEffect(() => {
  const unsubscribe = navigation.addListener('tabLongPress', (e) => {
    // Do something
  });

  return unsubscribe;
}, [navigation]);
```

Reply:
Yeah sure!!

Reply:
@okwasniewski 
I was trying to test this as suggested by you?
But I found that even **tabPress** is not working in this way.

```
import { useNavigation } from '@react-navigation/native';

React.useEffect(() => {
  const unsubscribe = navigation.addListener('tabPress', (e) => {
    // Do something
  });

  return unsubscribe;
}, [navigation]);
```

Is this expected?

Reply:
Great! Can you use it inside of the test app?

Reply:
Do you mean example app?

Reply:
Yes, to replace 

```
screenListeners={{
        tabLongPress: (data) => {
```

With the listeners inside of screens